tutorials/markdown/raw.md

```diff
@@ -0,0 +1,27 @@
+# Raw Framework Quickstart Guide
+
+This is a quick and dry explanation of how to use TableTop.JS to its full effectiveness. 
+For a softer introduction, check out our Checkers and SimpleGame tutorials. 
+
+## (yourgamehere).js file 
+
+This is your main game file. Here, you build the important components and start the animation rendering. All main game files will basically look the same, except you'll be substituting in your subclasses for the defaults as needed. 
+
+The main components are the Board, Game, Players, TurnMap, and View. You'll need to subclass at least the Board, Game, and View. For basic games, you'll be able to use the default Player class, as well as the default TurnMap. More advanced logic can be implemented in a custom TurnMap (see our MonopolyTurn file to see how to create a game of that complexity) and custom player objects. 
+
+
+## The Board 
+
+For your custom Board class, there's two main things you'll have to do. First, you need to build tiles for your game (using new TableTop.Tile()) and store them in this.tiles. Secondly, you'll likely need to build tokens for your game (using new TableTop.Token()). In checkers, tokens are the pieces. In tic-tac-toe, tokens are the Xs and Os. Again, these should be stored in this.tokens.
```

Comment:
Maybe something about the different board types? And that tokens are any object that goes on the board that's also associated w/ a player?